app/index.js

```diff
@@ -7,11 +6,11 @@ const TwoByTwoCards = React.createClass({
   render() {
     return (
       <View style={[styles.container, {flexDirection: 'row' }]}>
```

Comment:
**<a href='http://eslint.org/docs/rules/no-sequences'>no-sequences</a>**: Unexpected use of comma operator.